### PR TITLE
Support for Custom Images

### PR DESCRIPTION
**Pull request by alakae**
Added a resource for cloudscale.ch Custom Images and make it possible for servers to use is.

### CODE REVIEW

File: cloudscale/resource_cloudscale_custom_image.go [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+			Required: true,
+			ForceNew: true,
+		},
+		"name": {
+			Type:     schema.TypeString,
+			Required: true,
+		},
+		"slug": {
+			Type: schema.TypeString,
+			Required: true,
```
Comment (davidhalter) [inline cloudscale/resource_cloudscale_custom_image.go]: The slug is not required.

File: website/docs/r/custom_image.html.markdown [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+layout: "cloudscale"
+page_title: "cloudscale.ch: cloudscale_custom_image"
+sidebar_current: "docs-cloudscale-resource-custom-image"
+description: |-
+  Provides a cloudscale.ch Custom Image resource. This can be used to create, modify, and delete Custom Images.
+---
+
+# cloudscale\_custom_image
```
Comment (davidhalter) [inline website/docs/r/custom_image.html.markdown]: There's probably one escape missing, right?

**Comment by davidhalter** [comment]
Only saw the issues above, otherwise LGTM!

File: website/docs/r/custom_image.html.markdown (hunk 2 above)
Comment (alakae) [reply website/docs/r/custom_image.html.markdown]: According to [this tool](https://registry.terraform.io/tools/doc-preview) the backslash is probably not even required. But adding it for consistency with other docs.

File: cloudscale/resource_cloudscale_custom_image.go (hunk 1 above)
Comment (alakae) [reply cloudscale/resource_cloudscale_custom_image.go]: Good catch. thnx.

**Pull request merged**